# GetTargetPosition.srv
---
float64 target_x
float64 target_y
bool success